_pages/en_US/installing-hiyacfw.md

```diff
@@ -57,6 +57,7 @@ This section is entirely optional. If Unlaunch is enough for you, you can stop h
 1. Install Python 3 using your package manager if its not already installed
 1. Download the latest Python version of [hiyaCFW Helper](https://github.com/mondul/HiyaCFW-Helper/releases)
 1. Extract the hiyaCFW Helper archive anywhere on your PC
+1. Make sure the neccessary Python packages are installed running `pip3 install -r requirements.txt` in the directory containing the extracted files
```

Comment:
```suggestion
1. Make sure the necessary Python packages are installed by running `pip3 install -r requirements.txt` in the directory containing the extracted files
   - You can install pip by running `python3 -m ensurepip`
```
```suggestion
1. Make sure the necessary Python packages are installed by running `pip3 install -r requirements.txt` in the directory containing the extracted files
   - You can install pip with the `python3-pip` package
```
Here's a couple suggestions of a couple minor typo fixes and for installing pip, I think the first is more universally compatible, but I'm not sure if that module is always included